halinuxcompanion/notifier.py

```diff
@@ -90,13 +90,17 @@ async def init(self, dbus: Dbus, api: API, webserverver: Server, companion: Comp
         :param dbus: The Dbus class abstraction
         """
         # Get the interface
-        self.interface = await dbus.get_interface("org.freedesktop.Notifications")
-        # Setup dbus callbacks
-        self.interface.on_action_invoked(self.on_action)
-        self.interface.on_notification_closed(self.on_close)
-
-        # Setup http server route handler for incoming notifications
-        webserverver.app.router.add_route('POST', '/notify', self.on_ha_notification)
+        interface = await dbus.get_interface("org.freedesktop.Notifications")
+        if interface is not None:
```

Comment:
I'd consider reversing this check and returning early (or even raising an error) if the interface is not available. For now everything else depends on it and there's no point in continuing the `Notifier`'s setup if it's not there.